tests/string/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,5 +1,29 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`multiline.php 1`] = `
+<?php
+$string = 'foo
+bar';
+$chart = '
+{
+  "chart": {
+    "caption": "Worlds Top 10 largest companies in 2017"
+  }
+}';
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+$string = 'foo
+bar';
+$chart =
+    '
```

Comment:
By the way: Do we expect this linebreak?

Reply:
@czosel no, looks bug, need create issue

Reply:
Done! #461